SPECS/libguestfs/libguestfs.spec

```diff
@@ -61,6 +54,7 @@ BuildRequires:  acl
 BuildRequires:  attr
 BuildRequires:  augeas-devel >= 1.7.0
 BuildRequires:  augeas-libs
+BuildRequires:  ocaml-augeas-devel
```

Comment:
It looks like this is now listed twice - this can probably be removed.

Reply:
Removed the top one to follow the alphabet order